prow/config.yaml

```diff
@@ -3691,9 +3493,7 @@ presubmits:
         image: gcr.io/k8s-testimages/kubekins-e2e:latest-experimental
         imagePullPolicy: Always
         name: ""
-        resources:
-          requests:
-            memory: 6Gi
+        resources: {}
```

Comment:
we dropped the 6Gi to standardize this job to look like others?

Reply:
er no, these are auto generated (pull-security-kubernetes-*). they run in a different build cluster though so the resource request isn't super important. Need to check the source job though.

Reply:
done, fixed the source job and regenerated. I need to come back through and add better resource requests for all of these jobs in the very near future though.

Reply:
👍